### ISSUE TITLE
Does not start on Ubuntu 20.04.6 LTS

### ISSUE THREAD

**Issue by LeKovr** [didikz/gosysmon-web#2]
```
$ go run ./cmd/main.go
Starting system monitor
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x6c68b2]

goroutine 20 [running]:
main.main.func1(0xc000188000)
	/opt/src/github/gosysmon-web/cmd/main.go:72 +0xd52
created by main.main in goroutine 1
	/opt/src/github/gosysmon-web/cmd/main.go:26 +0xa5
exit status 2
```

**Comment by LeKovr** [comment]
The same on Ubuntu 22.04.5 LTS

**Comment by didikz** [comment]
Already fixed on #3
